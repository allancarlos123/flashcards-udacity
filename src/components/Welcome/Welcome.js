import React from 'react'
import { StyleSheet, Text, View } from 'react-native';

const Welcome = () => (
  <View style={styles.header}>
    <Text style={styles.title}>Welcome!</Text>
    <Text style={styles.subtitle}>Memorize anything</Text>
  </View>
)

const styles = StyleSheet.create({
  title: {
    fontSize: 32,
    color: '#ED4E5F',
    fontWeight: 'bold',
    paddingBottom: 10,
  },
  subtitle: {
    fontSize: 20,
    color: '#ED4E5F',
    paddingBottom: 10,
  }
});

export default Welcome