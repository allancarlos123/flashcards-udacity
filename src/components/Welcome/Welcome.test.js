import React from 'react'
import renderer from 'react-test-renderer'

import Welcome from './'

describe('Welcome', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Welcome />).toJSON();
    expect(tree).toMatchSnapshot();
  });
})