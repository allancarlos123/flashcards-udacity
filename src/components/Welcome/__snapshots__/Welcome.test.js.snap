// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome renders correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#ED4E5F",
        "fontSize": 32,
        "fontWeight": "bold",
        "paddingBottom": 10,
      }
    }
  >
    Welcome!
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#ED4E5F",
        "fontSize": 20,
        "paddingBottom": 10,
      }
    }
  >
    Memorize anything
  </Text>
</View>
`;
